### PR TITLE
LiveRadio Refactor Regarding Ares Response

### LINKED ISSUE #6134: On Demand Radio Page has correct ATI page metadata
...so that we can generate reports and insights that help us improve our reach and engagement.

- [x] _ATI metadata for the media player has already been completed as part of #5984_ 
- [x] Ensure that the correct ATI metadata for the page is present as per requirements [#WSMEDIA-155 - ATI Page Load Tracking ](https://jira.dev.bbc.co.uk/browse/WSMEDIA-155)
 - [ ] Automated tests to ensure that ATI metadata is as expected. Determine whether this needs to be unit, integration or E2E test.

**Additional Information:**
https://paper.dropbox.com/doc/On-Demand-Audio-SEO-Metadata--AxvL5ADACVHXwYNBm7Ehjo~tAg-9ccFbhwazDB56TYzhSZVP#:uid=546747479559112509242594&h2=ATI-Analytics

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Preparation for #6134

**Overall change:**
Refactoring to only pass the required values to page components instead of the full Ares response

**Code changes:**

- Stripped down the Ares response to pass along only necessary values
- Refactored LiveRadio ATI Analytics to handle new data format
- Refactored Chartbeat to handle new data format
- Renamed RadioPage to LiveRadioPage

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/integration/pages/liveRadioPage/__snapshots__/korean.test.js.snap [hunk 1]
```diff
@@ -1395,7 +1395,7 @@ exports[`Given I am on the Korean Live Radio page When I view the source code in
...
+      window.SIMORGH_DATA={"status":200,"pageData":{"content":{"blocks":[{"text":"BBC 코리아 라디오","markupType":"plain_text","type":"heading","uuid":"mock-uuid"},{"text":"세계와 한반도 뉴스를 공정하고 객관적으로 전달해 드립니다","type":"paragraph","uuid":"mock-uuid"},{"id":"mock-id","subType":"primary","format":"audio","externalId":"bbc_korean_radio","duration":"PT0S","caption":"","embedding":false,"available":true,"live":true,"type":"version","uuid":"mock-uuid"}]},"language":"ko","id":"mock-id","name":"BBC 코리아 라디오","summary":"세계와 한반도 뉴스를 공정하고 객관적으로 전달해 드립니다","pageTitle":"BBC 코리아 라디오 - BBC News 코리아","contentType":"player-live","pageIdentifier":"korean.bbc_korean_radio.liveradio.page"},"path":"/korean/bbc_korean_radio/liveradio","timeOnServer": "mock-time"}
```
Comment (jroebu14) [inline src/integration/pages/liveRadioPage/__snapshots__/korean.test.js.snap:1402]: oh wow. didn't realise this strategy would also cut down on the size of this object that hydrates the page on the client. For some pages it's absolutely huge. this is cool 😄

**Comment by rebeccamcginn** [comment]
Chartbeat requires further testing on AMP

**Review by jroebu14 — COMMENTED** [review]
looking good!

**Comment by karinathomasbbc** [comment]
Nice 👍

**Review by karinathomasbbc — APPROVED** [review]

**Comment by karinathomasbbc** [comment]
Oh, one thought I just had - will this log statement still output WS-LIVE -> https://github.com/bbc/simorgh/blob/eaaeed4db56784596ebb82c4ea032f58da1b1292/src/server/index.jsx#L283 - we use this in SumoLogic.

I expect the see the following output in the logs:
```
2020-04-17 14:07:45.257 info [server/index.jsx] {
  "event": "routing_info",
  "message": {
    "url": "/uzbek/bbc_uzbek_radio/liveradio",
    "status": 200,
    "pageType": "WS-LIVE"
  }
}
```

**Comment by rebeccamcginn** [comment]
> Oh, one thought I just had - will this log statement still output WS-LIVE ->
> 
> https://github.com/bbc/simorgh/blob/eaaeed4db56784596ebb82c4ea032f58da1b1292/src/server/index.jsx#L283
> 
> - we use this in SumoLogic

No, it won't get the pageType for liveradio or onDemand radio

**Comment by karinathomasbbc** [comment]
> No, it won't get the pageType for liveradio or onDemand radio

😢

**Review by ryanmccombe — APPROVED** [review]

**Pull request merged**